### PR TITLE
Add missing Connector methods

### LINKED ISSUE #9320: Add etag, checksum and last_modified methods to Connector class
Class `Connector` that handles connector imports and synchronizations is a substitute for the class `Importer2::Runner` which is used in other kind of imports/syncs. So it has to provide the same public interface as `Runner`, but some methods were missing.

This hasn't been noticed before as the missing methods are used after a successful sync, and the lack of them causes exceptions that don't transcend to errors visible in the UI.

After a sync is finished successfully, `Synchronization::Member#set_success_state_from`
collects information about it and in particular calls the methods `etag`, `checksum` and `last_modified` of `Connector` (via `Synchronization::Adapter`).  `Runner` obtains this information from the `Downloader`

### PR DESCRIPTION
**Pull request by jgoizueta**
Fixes #9320

### CODE REVIEW

**Comment by jgoizueta** [comment]
@juanignaciosl can you CR this?
cc @rafatower

**Comment by jgoizueta** [comment]
Note: note having nil values for etag/checksum/last_update doesn't seem a problem, as we have no meaningful values for connectors since we don't use downloads.

**Comment by juanignaciosl** [comment]
Two minor comments:

1. Not even `last_modified` value is needed?
2. As finding empty methods is confusing, please add comments to them explaining the need (your comment in the issue is nice, for example) so the next reader (or my future me :_) ) understand them.

👍

**Comment by jgoizueta** [comment]
I'm my tests, the lack of `last_modified` data didn't cause any problem.

I agree to place some comments in the methods :)

File: services/importer/lib/importer/connector.rb [hunk 1]
```diff
@@ -106,6 +108,21 @@ def self.check_availability!(user)
         end
       end
 
+      def etag
+        # This method is needed to make the interface of Connector compatible with Runner,
+        # but we have no meningful data to return here.
```
Comment (rafatower) [inline services/importer/lib/importer/connector.rb]: typo: meaningful

Comment (jgoizueta) [inline services/importer/lib/importer/connector.rb]: oops

**Pull request merged**